jets/e/argon2.c

```diff
@@ -109,7 +146,12 @@
     }
 
     u3z(key); u3z(extra); u3z(dat); u3z(sat);
-    return u3kc_rev(3, out, u3i_bytes(out, outhash));
+
+    // Copy result from the stack.
+    u3_noun result = u3i_bytes(out, output_buf);
+
+    // XX Do we need to free `result`?
+    return u3kc_rev(3, out, result);
```

Comment:
It's implicitly `u3z`'d, because `u3k*` jets have transfer semantics.

Reply:
Ah! Ok, that makes sense. Thanks.